Automattic/wp-calypso
# Update the appearance of the "Add a domain" button

#### Changes proposed in this Pull Request
We want to update the appearance of the "Add a domain" button at the top of the Site Domains page, as specified in: 1146722293412575-as-1200601027531795.

#### Testing instructions
Use the "Link to Calypso live" link below to view the build for this branch.
Go to the site domains page (My Sites -> Manage -> Domains).
Make sure that the button looks like the "New" version shown in the screenshot below:

Old button:
![image](https://user-images.githubusercontent.com/18705930/125891800-85bee448-53d6-4143-a99a-1cb1b004edd9.png)


New button:
![image](https://user-images.githubusercontent.com/18705930/125891777-3057c7a4-eb59-4ed3-b41f-38dcbd5b8251.png)
